### PR TITLE
CP-5514: Hotfix Make issues

### PR DESCRIPTION
**Pull request by Tankanow**
## Description of the change

> Hotfix Make issues

## Type of change
- [x] Bug fix
- [ ] New feature

### Development
- [x] All changed code has 80% unit test coverage
- [x] All changed code has been automatically (smoke test or otherwise) or manually verified in `alfa` (or with a cross namespace setup, e.g. developer namespace for this feature, pointing at shared `alfa` resources)

### Code review 
- [x]  This pull request has a title that includes the ticket # and a short useful summary, e.g. `CP-4051: Create TEMPLATE Feature Repo`.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -62,25 +65,39 @@ $(PYTHON_DEPENDENCY_FILE): $(REQUIREMENTS_FILES)
...
-.PHONY: lint-all $(ALL_CFN_TEMPLATES)
+.PHONY: lint-all-templates
+lint-all-templates: $(ALL_CFN_TEMPLATES)
+	cfn-lint -i W2001 -t $?
 
 
-test: lint test-all-apps																		## Lints then tests code for all available runtimes
-.PHONY: test
+.PHONY: lint-sam-apps
+lint-sam-apps: $(SAM_APP_LINT_RESULTS)
```
Comment (Tankanow) [inline Makefile:78]: NOTE: Ensure we have all targets

File: MakefileConstants.mk [hunk 2]
```diff
@@ -23,6 +23,8 @@ PYTHON_DEPENDENCY_FILE := .cz_py_dependencies_installed
 CFN_LINT_OUTPUT := cfn-lint.output
 
 # CFN Constants
+COVERAGE_XML := coverage.xml
```
Comment (Tankanow) [inline MakefileConstants.mk:26]: NOTE: Useful constants

File: services/account_type/master_payer.yaml [hunk 3]
```diff
@@ -15,6 +15,12 @@ Parameters:
     AllowedValues:
       - 'true'
       - 'false'
+  IsOrganizationMasterAccount:
```
Comment (Tankanow) [inline services/account_type/master_payer.yaml:18]: NOTE: Need this for CFN "api", that is parent stack expects this in all child stacks even if it's not used

File: Makefile [hunk 4]
```diff
@@ -62,25 +65,39 @@ $(PYTHON_DEPENDENCY_FILE): $(REQUIREMENTS_FILES)
...
+$(SAM_APP_ZIPS):
+	cd $(@D) && $(MAKE) package
 
 
-test-all-apps: $(SAM_APPS)
-$(SAM_APPS):
-	$(MAKE) test -C $@
-.PHONY: test-all-apps $(SAM_APPS)
+.PHONY: clean-sam-apps
+clean-sam-apps:
```
Comment (Tankanow) [inline Makefile:96]: NOTE: Probably a way to do this without a for loop, but punting on it for now b/c it's just the clean step

**Review by silvexis — APPROVED** [review]
Makefile master bringing it home

**Pull request merged**